internal/pkg/ecs/errors.go

```diff
@@ -12,3 +12,17 @@ type ErrMultipleContainersInTaskDef struct {
 func (e *ErrMultipleContainersInTaskDef) Error() string {
 	return fmt.Sprintf("found more than one container in task definition: %s", e.taskDefIdentifier)
 }
+
+type ErrExitCode struct {
```

Comment:
Does this need to be public? If so could you add comment to the public struct?

Reply:
Added a comment